utils/args/hcpfunc_qc_mosaic.py

```diff
@@ -3,43 +3,48 @@
 /tmp/scripts/hcpfunc_qc_mosaic.sh
 part of the hcp-func gear
 """
-import os, os.path as op
+import logging
+import os
+import os.path as op
 from collections import OrderedDict
+
 from .common import build_command_list, exec_command
-import logging
 
 log = logging.getLogger(__name__)
 
+
 def build(context):
     config = context.config
     params = OrderedDict()
     params['qc_scene_root'] = op.join(context.work_dir, config['Subject'])
     params['fMRIName'] = config['fMRIName']
     # qc_image_root indicates where the images are going
     params['qc_image_root'] = op.join(
-        context.work_dir, 
+        context.work_dir,
         config['Subject'] + '_{}.hcp_func_QC.'.format(config['fMRIName'])
     )
     context.gear_dict['QC-Params'] = params
 
+
 def validate(context):
     pass
 
```

Comment:
If these validate functions are not intended to be used in the final gear, I would recommend removing them. It's unnecessary code.

Reply:
Agreed. 

They are there as placeholders in case there was some validation that was necessary... and, truth be told, in some of these algorithms we have yet to know if there are some parameter conflicts. These parameter conflicts are not caught by the single-parameter validation of the UI.

Easy come. easy go.  And easy come again, if they are indeed needed.